commands/audit/audit.go

```diff
@@ -277,3 +277,26 @@ func downloadAnalyzerManagerAndRunScanners(auditParallelRunner *utils.SecurityPa
 	}
 	return
 }
+
+// This function checks if partial results is allowed. If so we log the error and continue.
+// If partial results is not allowed we add the error to the SecurityParallelRunner error's channel if one is provided, or simply return the error if not.
+func createErrorIfPartialResultsDisabled(auditParams *AuditParams, auditParallelRunner *utils.SecurityParallelRunner, extraMassageForLog string, err error) error {
```

Comment:
```suggestion
func createErrorIfPartialResultsDisabled(partialResultEnabled bool, auditParallelRunner *utils.SecurityParallelRunner, extraMassageForLog string, err error) error {
```
IMO, no need to pass `auditParams *AuditParams`, just the bool is ok